tests/wpunit/MediaItemMutationsTest.php

```diff
@@ -756,6 +756,83 @@ public function testCreateMediaItemMutation() {
 		$this->assertEquals( $expected, $actual['data'] );
 	}
 
+	/**
+	 * This function tests the createMediaItem mutation using a local file.
+	 *
+	 * @source wp-content/plugins/wp-graphql/src/Type/MediaItem/Mutation/MediaItemCreate.php
+	 * @return void
+	 */
+	public function testCreateMediaItemWithFileMutation() {
+		wp_set_current_user( $this->admin );
+
+		// Set our local file.
+		$this->create_variables['input']['file']     = $this->file;
+		$this->create_variables['input']['slug']     = 'test-medium';
+		$this->create_variables['input']['filePath'] = null;
+
+		/**
+		 * Create the createMediaItem
+		 */
+		$actual = $this->createMediaItemMutation();
+
```

Comment:
https://github.com/wp-graphql/wp-graphql/actions/runs/3033573100/jobs/4930710142#step:7:64742

once the bug itself if fixed, we should `$this->assertArrayNotHasKey( 'errors', $actual )`